File: ios/Views/WebengagePersonalizationViewManager.m
#import <React/RCTViewManager.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(WebengagePersonalizationViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(propertyId, NSInteger)
RCT_EXPORT_VIEW_PROPERTY(screenName, NSString)
RCT_EXPORT_VIEW_PROPERTY(width, CGFloat)
RCT_EXPORT_VIEW_PROPERTY(height, CGFloat)

@end

File: ios/Views/WebengagePersonalizationViewManager.swift

@objc(WebengagePersonalizationViewManager)
class WebengagePersonalizationViewManager: RCTViewManager {

  override func view() -> (UIView) {
      let nativeView = WEHInlineView(frame: CGRect(x: 0, y: 0, width: 300, height: 200))

      return nativeView

  }

  @objc override static func requiresMainQueueSetup() -> Bool {
    return false
  }
}

